<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btNew">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu,Helvetica,sans-serif</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Nuevo</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Fecha visualizada:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Desde</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="deFechaMostradaDesde">
          <property name="displayFormat">
           <string>dd/MM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Hasta</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="deFechaMostradaHasta">
          <property name="displayFormat">
           <string>dd/MM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Tipo movimiento</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbTipoMovimiento">
          <item>
           <property name="text">
            <string>TODOS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>INGRESOS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EGRESOS</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Cuenta</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbCuentas"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lbTitle">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>42</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
color: #252424;
background-color:rgb(244, 244, 244);

padding:3px;
font: bold 14px;
font-family: Ubuntu, Helvetica, sans-serif;
border: 1px solid #BFBFBF;

border-bottom: 0px solid white;
border-top-left-radius: 3px ;
border-top-right-radius: 0px solid white;
border-bottom-right-radius: 0px ;

}</string>
              </property>
              <property name="text">
               <string>Libro Diario</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbItemsCount">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>42</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
color: rgb(147, 147, 147);
background-color:rgb(244, 244, 244);

padding:3px;
font: 10px;
font-family: Ubuntu, Helvetica, sans-serif;
border: 1px solid #BFBFBF;

border-top-left-radius: 0px solid rgb(244, 244, 244);
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;
border-bottom-left-radius: 0px;
border-bottom: 0px solid white;
}</string>
              </property>
              <property name="text">
               <string>0 items(s) listado(s)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="twItems">
          <property name="font">
           <font>
            <family>Ubuntu,Helvetica,sans-serif</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QHeaderView::section {
border-radius: 3px;
padding: 2px 4px 2px 4px;

/* Fuente */
color: #3A3939;
text-align: left;
font-family: Ubuntu, Helvetica, sans-serif;

background-color:rgb(244, 244, 244);
/*qlineargradient(spread:reflect, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(240, 240, 240, 255), stop:0.509174 rgba(250, 250, 250, 255), stop:0.963303 rgba(255, 255, 255, 255));*/
/* Bordes */
border-bottom: 1px solid #E3E3E3;
border-top: 1px solid #FFFFFF;
padding:6px;
border-top-left-radius: 0px;

}

QTableWidget{
font-family: Ubuntu, Helvetica, sans-serif;
/*font-size: 12px;*/
text-align: left;

border: 1px solid #BFBFBF;

}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::AnyKeyPressed</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideLeft</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>24</number>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbTitulo_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
color: #252424;
background-color:rgb(244, 244, 244);

padding:1px;
font: bold 14px;
font-family: Ubuntu, Helvetica, sans-serif;
border: 1px solid #BFBFBF;


border-top-left-radius: 6px;
border-top-right-radius: 6px;
border-bottom-left-radius: 0px ;
border-bottom-right-radius: 0px ;
}</string>
            </property>
            <property name="text">
             <string>Balance</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbBalance">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
color: black;
background-color: #BFBFBF;
font: bold 18px;
font-family: Ubuntu, Helvetica, sans-serif;
border: 1px solid #BFBFBF;
border-top-left-radius: 0px;border-top-right-radius: 0px;border-bottom-right-radius: 6px;border-bottom-left-radius: 6px;
}</string>
            </property>
            <property name="text">
             <string>$ 00,00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
